uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -44,7 +45,9 @@ define([
       this.postContactSelection = false; // flag to track if user is selected for enabling UI
       this.requestStatuses = {};
       this.statusBeforeEdit = {};
+      this.supportedFileTypes = '';
       this.today = Date.now();
+      this.userRemovedFile = false; // flag to track if user has removed file
```

Comment:
I'm not sure I understand the necessity for this variable. Why do you care if a user removed a file or not? Why is it important in order to decide what message to show to the user?

This looks like a very ad-hoc property